### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by utkarsh006**
- Created a README file for this repo for better readability of users.

## Screenshots
Screenshot (226)

### CODE REVIEW

**Comment by utkarsh006** [comment]
@chriscool Kindly Review.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+## git.github.io
+
+- This Repo contains Soc and Outreachy projects of past years.
```
Comment (chriscool) [inline README.md:3]: I might be Ok with adding a README.md, but it's not true that this repo is just about GSoC and Outreachy. I think the README should contain more or less what there is on: https://git.github.io/

Comment (utkarsh006) [reply README.md]: @chriscool Kindly Review Now.

**Comment by chriscool** [comment]
This LGTM now, merged! Thanks @utkarsh006!

**Pull request merged**